config.env

```diff
@@ -0,0 +1,29 @@
+
+# Specify the GCP Project and Bucket info here.
+GCLOUD_PROJECT = 'mwrite-a835'
+GCLOUD_BUCKET = 'mpr-research-data-uploads'
+
+# Specify the number of months backwards to search courses. Defaults to 4.
+NUMBER_OF_MONTHS = 4
+
+
+#The remaining variables do not have to be changed, unless file names or folders have been changed.
+
+#Default Credential JSON file names, ideally rename JSON files to match.
+#To connect to Django DB
+DB_KEY = 'database-research_ro.json'
+#To Connect to GCP Service Account
+GCP_KEY = 'gcpAccessKey.json'
+#Place Keys in the 'Credentials' folder.
```

Comment:
I might need to look more into the GCloud Auth methods, as the Auth methods right now want an actual JSON file path. 
I make a JSON key file temporarily and immediately destroy it after the connection is made, but this feels a bit janky and I would like to fix that.

Reply:
Oh, it requires a file?  If you can't find a way around that, then making the key file from the application is not such a bad thing.  There are Python library functions for creating temporary files, like in `/tmp` on Linux environments.  Since nothing other than our application will ever access that environment, I think it's safe to create the file if it doesn't already exist and leave it there for the next time the application runs.

Reply:
I added a suggestion that I think may work.  See comment below (https://github.com/tl-its-umich-edu/mpr-research-data/pull/3#discussion_r868788736).